services/console-proxy/server/src/com/cloud/consoleproxy/ConsoleProxy.java

```diff
@@ -379,7 +381,23 @@ private static void startupHttpCmdPort() {
         }
     }
 
-    public static void main(String[] argv) {
+    private static void startServerForNoVNC() {
+        try {
+            ConsoleProxyServerFactory factory = getHttpServerFactory();
+            if (factory == null) {
+                s_logger.error("Unable to load HTTP server factory");
+                System.exit(1);
+            }
+            Server webServer = factory.
```

Comment:
Fix formating, as long as code can be squeezed into >120 lines keep it on the same line